account/src/main/java/com/caldevsupplychain/account/controller/AccountControllerImpl.java

```diff
@@ -59,6 +59,13 @@
 	/************************************************************************************************
 	 |									Account API													|
 	 ************************************************************************************************/
+	@GetMapping("/users")
```

Comment:
Better to have permission here. 
Let's only allow ADMIN to read all users for now.